src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name();
```

Comment:
It's good that you're using the `name()` method to get the String representation of the enum constant. This is the preferred way over `toString()` as it guarantees the correct name of the enum constant.